### ISSUE TITLE
Has the app been abandonded? 

### ISSUE THREAD

**Issue by TobiasDev** [codinguser/gnucash-android#841]
Just wanted to ask if this is a temporary stop because you are busy with other things or if the app has been abandonded? :)

**Comment by azzamsa** [comment]
This is great app (sadly some part of it need a fix).

I hope the devs continue to maintain this project. Devs can open the donation page so we can keep this project alive.

**Comment by kbytesys** [comment]
The Gnucash website adverts this application, but if it has been abandoned it could be useful if they get the ownership of this project.

There is no way to contact the actual owner? Just to know his intentions.

**Comment by nottheend** [comment]
I don't think so.

There are commits

**Comment by nottheend** [comment]
Same thing has also been asked in #808

**Comment by kbytesys** [comment]
yes, but no answers and no github activity (not only for this project)
since 2018.

it's sound like the maintainer is m.i.a.

Il mer 25 set 2019, 18:45 nottheend <notifications@github.com> ha scritto:

**Comment by nottheend** [comment]
There was actually some kind if activity, e.g. closing #809 on 1 Mar 19.

But I agree, progress is very slow

**Comment by nottheend** [comment]
I'm thinking about forking this project as there is no response from repo owner

**Comment by Nebucatnetzer** [comment]
@nottheend I would say go for it. If he becomes active again there's still the possibility of merging any changes back to his project.

**Comment by nottheend** [comment]
I contacted the owner of the repo and he replied, that all pull requests will be merged.

Still waiting for https://github.com/codinguser/gnucash-android/pull/884 to be merged

**Comment by strixaluco** [comment]
@nottheend that's inspiring. How did you manage to reach the owner, if that's not a secret?

**Comment by nottheend** [comment]
@strixaluco just sent a mail to his public address and waitet several weeks

**Comment by strixaluco** [comment]
Thank you. @JeanGarf tried the same, but haven't received reply, so that's why I was curious.

**Comment by JeanGarf** [comment]
Hi everybody,

I added a page on my web site to allow downloading of the apk with all my contributions (my 10 Pull Requests)
http://thierryw777.free.fr/index.php/bureautique/comptabilite/gnucash-android/

If you're interested to test or use it, tell me if it works fine.

See you
